CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-08-07
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
```

Comment:
_linux host_
Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows 2022 hosts.